### DISCUSSION TITLE
Priyanshu's Introduction

### DISCUSSION THREAD

**Discussion by Priyanshu-CODERX** [community/community#21021] [Other Feature Feedback, Questions, & Ideas]
Hi! Everyone👋

I am Priyanshu Bhattacharjee, a 17 year old Extended Reality Engineer from India, having experience of more than 2 years in developing XR applications and have taught over 200+ students at different universities about XR, I am also Co Founder of Antern and also an YouTuber 😃.

My free 11 hours Augmented Reality Development course was also published on FreeCodeCamp's YouTube Channel 😀

It's a pleasure to meet you 😃

YouTube: https://www.youtube.com/c/CodedRealityXR
Linkedin: https://www.linkedin.com/in/priyanshu-bhattacharjee/
Instagram: https://www.instagram.com/priyanshucoderx/

**Comment by radar-blip** [comment]
Greetings from USA. 
Those are some awesome achievments you have going there.

**Comment by melvincwng** [comment]
@Priyanshu-CODERX , Hello there welcome to GitHub Community 👋

Nice to meet you 😊
